### ISSUE TITLE
Improve infrastructure for keeping track of sequences of missions

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#1916]
Right now I am working on forcing turkers into a specific mission flow (#1915). I am doing a sort of "quick fix" way to know when a turker should be switching from audit missions to validation missions and vice versa.

I think that in the future we should probably have a table that keeps track of sequences of missions. Right now I am just inferring what type of mission should be done next from context, which works fine because of the simple sequence of missions we are having users do right now. But this should be made more robust in the future.

I just need to think a bit harder about how to implement this, and I wanted to get our next turk deployment off the ground a bit faster than implementing this would require.

**Comment by misaugstad** [comment]
I am going to reference this issue number in a few places to help know what needs to be fixed in the future when implementing this. So when implementing, search the code base for "#1916".

**Comment by jonfroehlich** [comment]
Just pinging you on this @misaugstad to see if this should be a high(er) priority or if you want to continue to wait on this.

**Comment by misaugstad** [comment]
Nah, still low priority I think. The current structure is working okay. It's not super robust, and the sequence of missions doesn't always work out exactly as intended, but it isn't going so bad that this needs to be changed rn.

**Comment by misaugstad** [comment]
Closing this as we're going to be removing the MTurk support.
